src/main/java/seedu/address/model/person/Priority.java

```diff
@@ -0,0 +1,82 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's priority number in the client list.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPriority(String)}
+ */
+public class Priority {
+
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Priority numbers should only contain characters L,M,H,U OR ulmh";
```

Comment:
Perhaps we should change to
"Priority values should only take L, M, H, U, l, m, h, u, low, medium, high, or undefined;